j2/vars/globals.yaml

```diff
@@ -578,6 +594,7 @@ schema:
         - name: Credential_Principal
         - name: Credential_Secret
         - name: Description
+          index: false

```

Comment:
Sorry to go back and forth on this one; let's remove the index: false from here as well, as we may run into issues no 'Snowy Peak' :)